src/Terrain/MapMaster.java

```diff
@@ -68,6 +72,17 @@ public MapMaster(CommandingOfficers.Commander[] COs, MapInfo mapInfo)
           System.out.println("Warning! CO specified as owner of an uncapturable location in map " + mapInfo.mapName);
         }
       }
+      
+      if( mapInfo.mapUnits.size() > co )
```

Comment:
Uhhh, we can log something, but enforcing it will require other changes; the built in maps don't have any predeployed units, so they don't have anything in the unit map.

Reply:
Specifically...
The constructor overload for MapInfo would have to manually put elements into that list, or the things calling it updated to call the other one.
Running a for loop to set up something we don't care about since it's deliberately empty seems silly.
(I tried setting the starting capacity, and that obviously didn't work 'cause it's an ArrayList)

Reply:
I suggested to enforce that size = COs.length only if size is non-zero. A zero-size array would be fine in this case.

Reply:
....I guess I'm blind then, aren't I? Oh well.
Fair enough.